harness/determined/exec/tensorboard.py

```diff
@@ -88,5 +87,65 @@ def still_alive() -> bool:
     return p.wait()
 
 
+def get_tensorboard_version(version: str) -> Tuple[str, str]:
+    """
+    Gets the version of the tensorboard package currently installed. Used
+    by downstream processes to determine args passed in.
+    :return: version in the form of (major, minor) tuple
+    """
+
+    major, minor, _ = version.split(".")
+
+    return major, minor
+
+
+def get_tensorboard_args(args: List[str]) -> List[str]:
+    """
+    Builds tensorboard startup args from args passed in from tensorboard-entrypoint.sh
+    Args are added and deprecated at the mercy of tensorboard; all of the below are necessary to
+    support versions 1.14, 2.4, and 2.5
+    """
+    task_id = os.environ["DET_TASK_ID"]
+    port = os.environ["TENSORBOARD_PORT"]
+
+    # Version is passed in from tensorboard-entrypoint.sh which determines the version of
```

Comment:
This comment might not be completely necessary as I can understand what you're popping by the variable name. At the very least, try to make it as concise as possible (<= 1 line)